app.rb

```diff
@@ -0,0 +1,179 @@
+require_relative './student'
+require_relative './book'
+require_relative './rental'
+require_relative './teacher'
+require_relative './classroom'
+
+# rubocop:disable Metrics
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @class = Classroom.new('Grade 5')
+  end
+
+  def run
+    puts 'Welcome to School Library App!'
+    puts
+    select
+  end
+
+  def select
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    option = gets.chomp
+    run_choice option
+  end
+
+  def run_choice(input)
+    case input
+    when '1'
+      list_books
+    when '2'
+      list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
+    when '7'
+      puts 'See you again'
+    else
+      puts 'Invalid! Please enter a number from 1 to 7'
+      select
+    end
+  end
```

Comment:
- [ ] Please 🙏🏽 create a main.rb file as requested in the project to approve it 👇🏽 

> In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
> Present the user with a list of options to perform.
> Lets users choose an option.
> If needed, ask for parameters for the option.
> Have a way to quit the app.